README.md

```diff
@@ -296,6 +296,10 @@ to `False`, and `CLUSTER_KEY` to what you wish to use as the cluster name.
 By default, the Snowflake database is set to `PROD`. To override this, set `DATABASE_KEY`
 to `WhateverNameOfYourDb`.
 
+By default, the Snowflake schema is set to `INFORMATION_SCHEMA`. To override this, set `SCHEMA_KEY`
+to `WhateverNameOfYourSchema`. Note that https://docs.snowflake.com/en/sql-reference/account-usage.html#differences-between-account-usage-and-information-schema `ACCOUNT_USAGE` is a separate schema
+which allows users to query a wider set of data at the cost of latency.
```

Comment:
Did you mean to have the URL as a link for text `ACCOUNT_USAGE`.

Reply:
Ah good catch, just updated the wording a bit